test/execute.js

```diff
@@ -441,7 +441,7 @@ describe('execute', () => {
       })
     })
 
-    it('should handle requestContentType', function () {
+    it('should not add content-type with no form-data or body param', function () {
```

Comment:
With the issue in https://github.com/swagger-api/swagger-ui/issues/2909, there is likely a `requestContentType` being passing in from swagger-ui. And that shouldn't happen